http-gateway/web/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "plgd-dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@auth0/auth0-react": "^1.2.0",
+    "@babel/cli": "^7.12.10",
+    "@craco/craco": "^6.0.0",
+    "@reduxjs/toolkit": "^1.5.0",
+    "@testing-library/jest-dom": "^5.11.8",
+    "@testing-library/react": "^11.2.2",
+    "@testing-library/user-event": "^12.6.0",
+    "bootstrap": "^4.5.3",
+    "classnames": "^2.2.6",
+    "cross-env": "^7.0.3",
+    "history": "^4.10.1",
+    "node-sass": "4.14.1",
+    "prettier": "1.14.3",
+    "react": "^17.0.1",
+    "react-bootstrap": "^1.4.0",
+    "react-dom": "^17.0.1",
+    "react-intl": "^5.10.11",
+    "react-intl-po": "^2.2.2",
```

Comment:
This is a very difficult decision to make actually, since most of these should be devDependencies, but I found out that there are different opinions about it. For instance, in production, you should call the npm install with the production flag `npm install --production` which won't install the devDependencies. I'll keep everything in dependencies for now, will separate these later.